### PR TITLE
Updated falixnodes on providers list

### PR DESCRIPTION
**Pull request by AgentDeath253**
Updated falixnodes

### CODE REVIEW

**Review by onebeastchris — CHANGES_REQUESTED** [review]
Maybe it would be worth it to instead say "change the 'port' under the 'bedrock' section", as there are 2 port entries in the config.
Also,  punctuation at the end of the sentence wouldn't hurt

**Comment by AgentDeath253** [comment]
> Maybe it would be worth it to instead say "change the 'port' under the 'bedrock' section", as there are 2 port entries in the config. Also, punctuation at the end of the sentence wouldn't hurt
> 
> Also, "Geyser" with a capital "G"

better?

**Comment by onebeastchris** [comment]
Yea, although 'Geyser MC Configuration' sounds a bit.. odd
maybe just "in the Geyser config."?

**Comment by onebeastchris** [comment]
thanks, that's better

sorry if i'm annoying, but please do add a "." at the end of the last sentence - other than that, this looks good. Also, do users need to specify whether the port is for UDP/TCP?

**Comment by AgentDeath253** [comment]
do users need to specify whether the port is for UDP/TCP?

no they do not need to specify if the port is UDP/TCP

**Review by onebeastchris — APPROVED** [review]

File: _data/providers.json [hunk 1]
```diff
@@ -170,7 +170,7 @@
     {
       "name": "FalixNodes",
       "url": "https://falixnodes.net/",
-      "description": "Get Geyser as a plugin. Need to ask to open a UDP port."
+      "description": "Get Geyser as a plugin. Open a port yourself on the network page in the game panel, then  change the port under the bedrock section of the Geyser config."
```
Comment (rtm516) [inline _data/providers.json:173]:
```suggestion
      "description": "Get Geyser as a plugin. Open a port yourself on the network page in the game panel, then change the port under the bedrock section of the Geyser config."
```

**Review by rtm516 — CHANGES_REQUESTED** [review]

**Review by onebeastchris — APPROVED** [review]

**Review by rtm516 — APPROVED** [review]

**Pull request merged**